machines/pod_X1Y1_ruche_X16Y8_hbm/Makefile.machine.include

```diff
@@ -0,0 +1,125 @@
+# This is probably not a sensible configuration, as the tile to channel ratio is very low. #
+# But for memory bound benchmarks, it may be reasonable to use for iteration.              #
+############################################################################################
+
+# Change these parameters to define your machine. All other parameters should remain constant.
+
+# Chip Pod Dimensions
+BSG_MACHINE_PODS_X                    = 1
+BSG_MACHINE_PODS_Y                    = 1
+
+# Heterogeneous tile composition. Must be a 1-d array equal to the
+# number of tiles, or shorthand (default:0)
+# 0 is for Vanilla Core (RV32).
+# Anything else should be described here :)
+BSG_MACHINE_HETERO_TYPE_VEC           = default:0
+
+# Network Parameters
+BSG_MACHINE_NETWORK_CFG               = e_network_half_ruche_x
+BSG_MACHINE_RUCHE_FACTOR_X            = 3
+
+# Memory System Parameters
+BSG_MACHINE_DRAM_INCLUDED             = 1
+BSG_MACHINE_MEM_CFG                   = e_vcache_hbm2
+
+# Victim Cache Parameters
+BSG_MACHINE_VCACHE_PER_DRAM_CHANNEL   = 16
+BSG_MACHINE_VCACHE_SET                = 64
+BSG_MACHINE_VCACHE_WAY                = 8
+BSG_MACHINE_VCACHE_LINE_WORDS         = 32
+BSG_MACHINE_VCACHE_STRIPE_WORDS       = $(BSG_MACHINE_VCACHE_LINE_WORDS)
+# Only applies to Non-blocking Cache
+BSG_MACHINE_VCACHE_MISS_FIFO_ELS      = 32
+
+# NOC Coordinate Widths (Rarely Changed)
+BSG_MACHINE_NOC_COORD_X_WIDTH         = 7
+BSG_MACHINE_NOC_COORD_Y_WIDTH         = 7
+
+# Pod Tile Dimensions (Rarely Changed)
+BSG_MACHINE_POD_TILES_X               = 16
+BSG_MACHINE_POD_TILES_Y               = 8
+
+# Host Coordinate (Rarely Changed)
+BSG_MACHINE_HOST_COORD_X              = 16
+BSG_MACHINE_HOST_COORD_Y              = 0
+
+BSG_MACHINE_HOST_X_CORD              = $(BSG_MACHINE_HOST_COORD_X)
+BSG_MACHINE_HOST_Y_CORD              = $(BSG_MACHINE_HOST_COORD_Y)
+
+# Manycore Origin (Rarely Changed)
+BSG_MACHINE_ORIGIN_COORD_X            = 16
+BSG_MACHINE_ORIGIN_COORD_Y            = 8
+
+# Cache-DRAM Interface Width (Rarely Changed)
+BSG_MACHINE_VCACHE_DMA_DATA_WIDTH     = 32
+
+# IO flow control parameters. (Rarely Changed)
+BSG_MACHINE_IO_EP_CREDITS             = 16
+BSG_MACHINE_IO_HOST_CREDITS           = $(BSG_MACHINE_IO_EP_CREDITS)
+BSG_MACHINE_IO_REMOTE_LOAD_CAP        = $(BSG_MACHINE_IO_EP_CREDITS)
+
+##################### Constants and Computations #####################
+
+# Aliases required for memsys.mk, and others.
+# Specific to the HBM2E chip we're modeling (8GB, 4-stack/8-channel)
+__BSG_MACHINE_DRAMSIM3_CHIP_SIZE_IN_WORDS := $(shell echo 2^31 | bc) # 4GB
+__BSG_MACHINE_DRAMSIM3_CHIP_CH_SIZE_IN_WORDS := $(shell echo $(__BSG_MACHINE_DRAMSIM3_CHIP_SIZE_IN_WORDS)/8 | bc)
+
+# TODO: Specific to pod (next 4)
+BSG_MACHINE_NUM_VCACHE               := $(shell echo $(BSG_MACHINE_POD_TILES_X)*2 | bc)
+BSG_MACHINE_DRAM_CHANNELS            := $(shell echo $(BSG_MACHINE_NUM_VCACHE)/$(BSG_MACHINE_VCACHE_PER_DRAM_CHANNEL) | bc)
```

Comment:
Need to fix

Reply:
Go ahead and fix if you know what to do.

Reply:
We need to enumerate the number of vcaches. This is only partially correct. Maybe it's as simple as multiplying by pod Y and X

Reply:
Fixed